rpcserver.go

```diff
@@ -3572,11 +3573,54 @@ func createRPCOpenChannel(r *rpcServer, graph *channeldb.ChannelGraph,
 	for i, htlc := range localCommit.Htlcs {
 		var rHash [32]byte
 		copy(rHash[:], htlc.RHash[:])
+
+		circuitMap := r.server.htlcSwitch.CircuitModifier()
+
+		var linkedChannel, linkedHtlcIndex uint64
+		switch {
+		case htlc.Incoming:
+			circuit := circuitMap.LookupCircuit(
+				htlcswitch.CircuitKey{
+					ChanID: dbChannel.ShortChannelID,
+					HtlcID: htlc.HtlcIndex,
+				},
+			)
+			if circuit != nil {
+				linkedChannel = circuit.Outgoing.ChanID.
```

Comment:
This is panicing in itests

Reply:
Hmm, I thought that the itests are also part of the list of github actions in CI. Quickly checked those and they were green. But  apparently need to always check Travis too then.

Reply:
They were moved out of actions a while ago because you can't restart an individual action, so one itest flake would mean that every action needs to be run again. While we wait on that feature in actions, it didn't make sense to run them twice.

Reply:
Ah ok, I missed that. 

Nil check added.